ts/parsers/paragraphparser.ts

```diff
@@ -118,6 +118,11 @@ export default class ParagraphParser {
                 i--;
             }
         }
+
+        if (list.listItems.length !== 1 && list.listItems[list.listItems.length - 1].list) {
```

Comment:
so negative values are OK or can this be `list.listItems.length > 1`

Reply:
I suppose your top level list is at index -1 so it is a specific case here

Reply:
changed it to `> 1`